controllers/argocd/keycloak.go

```diff
@@ -307,7 +308,7 @@ func getKeycloakContainer(cr *argoprojv1a1.ArgoCD) corev1.Container {
 			{ContainerPort: 8888, Name: "ping", Protocol: "TCP"},
 		},
 		ReadinessProbe: &corev1.Probe{
-			FailureThreshold: 10,
+			FailureThreshold: 20,
```

Comment:
This change is not so relevant to this PR but I noticed that sometimes keycloak might take more time to be ready or accept traffic.